woof-code/3builddistro

```diff
@@ -508,9 +508,11 @@ if [ -n "$DOCS_URL" ]; then
 	if [ "$CACHE_PERSONAL_DOCS" = 'y' ]; then
 		mkdir -p "../../local-repositories/personal_docs/$DOCS_URL"
 		DOWNLOADED_DOC_FILENAME="../../local-repositories/personal_docs/$DOCS_URL/doc.zip"
-		[ -f $DOWNLOADED_DOC_FILENAME ] || wget_output=$(wget -t 3 -T 60 -O $DOWNLOADED_DOC_FILENAME "$DOCS_URL")
+		[ -f $DOWNLOADED_DOC_FILENAME ] || wget -t 3 -T 60 -O $DOWNLOADED_DOC_FILENAME "$DOCS_URL"
+		wget_return_code=$?
```

Comment:
`wget_return_code` is undefined if `[ -f $DOWNLOADED_DOC_FILENAME ]`!

```
                wget_return_code=0         <--
		[ -f $DOWNLOADED_DOC_FILENAME ] || wget -t 3 -T 60 -O $DOWNLOADED_DOC_FILENAME "$DOCS_URL"
		wget_return_code=$?
```

Reply:
How can I fix this? Use `if` maybe?

Reply:
Either initialize it before the condition (see the example I gave you), or just use `if`

Reply:
I have used `if`. Seems like my bash scripting knowledge is still a trash :frowning_face:! Sorry for constant silly bugs.